src/configure/helper/remoteServiceUrlHelper.ts

```diff
@@ -26,10 +29,19 @@ export class RemoteServiceUrlHelper {
             } as IServiceUrlDefinition;
         }
 
-        return this.getServiceurlDefinition("https://peprodscussu2.portalext.visualstudio.com", this.templateServiceRedirectUrl);
+        if (deployment != undefined && deployment === "test") {
+            return this.getServiceurlDefinition("https://pepfcusc.portalext.visualstudio.com/_apis/TemplateService/", this.templateServiceStagingRedirectUrl);
```

Comment:
Looks like the first argument is pretty useless. Shall we remove it from the function  `getServiceurlDefinition` definition?

Reply:
It can be used to handle broken redirect URLs in that case this hardcoded link will be set. But the scenario is quite rare

Reply:
I will remove it

Reply:
done